src/constraints/create.jl

```diff
@@ -149,7 +171,7 @@ function _create_constraints_tables(connection)
     DuckDB.query(
         connection,
         "CREATE OR REPLACE TEMP SEQUENCE id START 1;
-        CREATE OR REPLACE TABLE cons_capacity_outgoing_binary AS
+        CREATE OR REPLACE TABLE cons_capacity_outgoing_storage_with_binary AS
```

Comment:
Took me a while, but if I understood correctly, this table should have a `WHERE` condition to check `NOT asset.investable` for it to be equal to the old code.

My logic (and I will use initials to avoid writing for 30 minutes) is that initially you had two tables: COB and COBI

- For COB, if investable, use `(c * aiu + ail) * (1 - ch)` else use `c * aiu * (1 - ch)`
- FOR COBI, use `c * (aiu * (1 - ch) + aiusm)`

Now you have three tables, COSB, COISB, COISLB.

- For COSB, `c * aiu * (1 - ch)`, so COSB = COB + not investable
- For COISB, `c * (aiu * (1 - ch) + aiusm)`, so COISB = COBI
- For COISLB, `(c * aiu + ail) * (1 - ch)`, so COISLB is COB + investable (which is COBI, I think)

If that is the case, you can also reuse the COISB table - you still separate the constraints to avoid the if condition, you just attach to the same constraint.
I checked the formulation, and indeed it separates like these tables, but I think that the formulation is also missing a `a \not \in A_y^I` or similar for that constraint.

Let me know if this makes sense or not

Reply:
It makes sense, I left a couple of comments where I see we need the changes and also a rename in one of the constraints to make it explicit.

Reply:
We need to update the formulation, too

Reply:
Thanks. Do you want to finish the PR with these changes?